app/views/govuk_publishing_components/components/docs/layout_footer.yml

```diff
@@ -123,3 +123,15 @@ examples:
             text: Rhestr o Wasanaethau Cymraeg
           - href: '/government/organisations/government-digital-service'
             text: Government Digital Service
+
+  with_new_windows:
```

Comment:
can we name this `whith_target_on_links` to make it more clear?

Reply:
Done